### ISSUE TITLE
Continuation

### ISSUE THREAD

**Issue by Arkosee** [MaDHouSe79/mh-inventory#3]
#2 in reference to your last post;

I've updated both this morning this happens to the inventory when opening it to pickup dropped items;
![Screenshot 2023-10-16 065615](https://github.com/MaDHouSe79/mh-inventory/assets/105973488/731cb33f-97fc-477c-b5fd-c5b5c5c7f18b)

This is the error relating to the blank drop side;

![Screenshot 2023-10-16 065908](https://github.com/MaDHouSe79/mh-inventory/assets/105973488/43e498c2-8c98-47d0-b5d5-25c686369f49)

After getting the "blank" drop side you have to close the inventory and re-open it to fix it then get this error when pulling things from the "drop" side of the inventory and the bag remains on ground

![Screenshot 2023-10-16 065733](https://github.com/MaDHouSe79/mh-inventory/assets/105973488/76b0ac6b-28eb-456c-8569-2e30a2097782)

Outside of that, seems to be working well :)

**Comment by MaDHouSe79** [comment]
how do you get 6000 lines? the main.lua file is about 3300 lines not 6000 :) i checkt is but i think i copy 2 times and not one time, copy the server file again from the inventory and check if it works now ;)

**Comment by Arkosee** [comment]
> how do you get 6000 lines? the main.lua file is about 3300 lines not 6000 :) i checkt, but i think i copy 2 times and not one time, copy the server file again from the inventory and check if it works now ;)


Nice, that fixed the error, but it does this weird thing I grabbed a vide of, if you split money (or anything that can stack I guess) it doesn't accept it if you pick it up and place it with the other items of its type. So you have to place it in a free slot, and then it'll auto combine it on close.. Not a huge issue but it's a little strange... There's no errors surrounding it though

[Quick video](https://www.youtube.com/watch?v=t_LpAXDC0S8)

**Comment by MaDHouSe79** [comment]
did you still have this issue as showing in the video?

**Comment by Arkosee** [comment]
> did you still have this issue as showing in the video?

Yes I do

**Comment by MaDHouSe79** [comment]
did you try to download a new version cause i already update it.

**Comment by Arkosee** [comment]
> did you try to download a new version cause i already update it.

Yep, I just reupdated it about an hour ago just to make sure, still does it

**Comment by MaDHouSe79** [comment]
i have a qb-inventory on my git can you see if that one works? 
thats a new update default qbcore qb-inventory that i just made for mh-cashasitem.

**Comment by Arkosee** [comment]
> i have a qb-inventory on my git can you see if that one works? thats a new update default qbcore qb-inventory that i just made for mh-cashasitem.
> 
> also make sure you have this in shared item file in cash and blackmoney
> 
> ['unique'] = false,

Doesn't fix anything except now I have a bunch of these,

![Screenshot 2023-11-04 211358](https://github.com/MaDHouSe79/mh-inventory/assets/105973488/65732955-8257-4b4d-9733-b7c8df74d513)
